packages/opentelemetry-node/test/instr-http.test.js

```diff
@@ -6,44 +6,86 @@
 // Test that 'http' instrumentation generates the telemetry we expect.
 
 const test = require('tape');
-const {runTestFixtures} = require('./testutils');
+const {runTestFixtures, filterOutGcpDetectorSpans} = require('./testutils');
 
 /** @type {import('./testutils').TestFixture[]} */
 const testFixtures = [
     {
-        name: 'http.get',
+        name: 'http.get (stable semconv as default)',
         args: ['./fixtures/use-http-get.js'],
         cwd: __dirname,
         env: {
             NODE_OPTIONS: '--require=@elastic/opentelemetry-node',
         },
         // verbose: true,
         checkTelemetry: (t, col) => {
-            const spans = col.sortedSpans;
+            const spans = filterOutGcpDetectorSpans(col.sortedSpans);
+            t.equal(spans.length, 1);
+            const span = spans[0];
+            t.equal(span.scope.name, '@opentelemetry/instrumentation-http');
+            t.equal(span.name, 'GET');
+            t.equal(span.kind, 'SPAN_KIND_CLIENT');
+            t.equal(span.attributes['http.url'], undefined);
+            t.equal(span.attributes['url.full'], 'http://www.google.com/');
+        },
```

Comment:
A test case that tests that the default `'http'` works when `OTEL_SEMCONV_STABILITY_OPT_IN` has other non-http-related values in it would be good. E.g. `OTEL_SEMCONV_STABILITY_OPT_IN=db/dup,foo,` or something